src/Mod/Fem/Gui/AppFemGui.cpp

```diff
@@ -176,8 +175,7 @@ PyMOD_INIT_FUNC(FemGui)
     new Gui::PrefPageProducer<FemGui::DlgSettingsFemElmerImp>(QT_TRANSLATE_NOOP("QObject", "FEM"));
     new Gui::PrefPageProducer<FemGui::DlgSettingsFemMystranImp>(QT_TRANSLATE_NOOP("QObject", "FEM"));
     new Gui::PrefPageProducer<FemGui::DlgSettingsFemZ88Imp>(QT_TRANSLATE_NOOP("QObject", "FEM"));
-    new Gui::PrefPageProducer<FemGui::DlgSettingsFemMaterialImp>(QT_TRANSLATE_NOOP("QObject", "FEM"));
-
+    
```

Comment:
Trailing whitespace

Reply:
See the rest of the trailing whitespaces in https://github.com/FreeCAD/FreeCAD/actions/runs/5983698946/job/16234124245?pr=10368#step:5:1